bot/bot.py

```diff
@@ -45,8 +45,7 @@ def __init__(self, *args, **kwargs):
             # will effectively disable stats.
             statsd_url = "127.0.0.1"
 
-        asyncio.create_task(self._create_redis_session())
-
+        self.loop.create_task(self._create_redis_session())
```

Comment:
Other "lazy" session creators hook into the `_recreate` method below to do their work. What's the reason for doing it here instead of there? I don't think I mind either way, this will ensure that it will be scheduled as soon as the loop starts.

The only thing I'm slightly worried about is that I don't know when Discord calls `Bot.close()`: If that's done before it calls `Bot.clear()`, your redis session will be closed but not recreated.

Edit: A quick glance at `discord.py` makes me think it won't do that latter thing. I don't think we need to recreate our Redis session often as well. Hopefully never.

Reply:
I've moved it over to `_recreate` anyway. I didn't do it originally because I just wasn't aware of it. Seems like a better fit for that method, as long as it works.